python/ray/autoscaler/cli_logger.py

```diff
@@ -513,6 +513,34 @@ def confirm(self, yes, msg, *args, _abort=False, _default=False, **kwargs):
 
         return res
 
+    def prompt(self, msg: str, *args, **kwargs):
+        """Prompt the user for some text input.
+
+        Args:
+            msg (str): The mesage to display to the user before the prompt.
+
+        Returns:
+            The string entered by the user.
+        """
+        if self.old_style:
+            return
+
+        complete_str = cf.blue(cf.underlined(msg))
+        rendered_message = _format_msg(complete_str, *args, **kwargs)
+        if rendered_message and rendered_message[-1] != "\n":
+            rendered_message += " "
+        self._print(rendered_message, linefeed=False)
```

Comment:
@maximsmol could you take another look at this section? I'm not really sure how to use `_format_msg` or `_print`, so I just guessed.

Reply:
lgtm